src/components/PostsList.tsx

```diff
@@ -1,56 +1,59 @@
 import classNames from 'classnames';
-import React from 'react';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 import { Post } from '../types/Post';
+import { set as setSelectedPost } from '../features/selectedPost';
 
-type Props = {
-  posts: Post[],
-  selectedPostId?: number,
-  onPostSelected: (post: Post | null) => void,
-};
-
-export const PostsList: React.FC<Props> = ({
-  posts,
-  selectedPostId = 0,
-  onPostSelected,
-}) => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+export const PostsList = () => {
+  const { items } = useAppSelector(state => state.posts);
+  const dispatch = useAppDispatch();
+  const selectedPost = useAppSelector(
+    state => state.selectedPost,
+  ) as Post | null;
```

Comment:
Looks like redundant
```suggestion
  const selectedPost = useAppSelector(state => state.selectedPost);
```